presto-base-jdbc/src/main/java/com/facebook/presto/plugin/jdbc/BaseJdbcClient.java

```diff
@@ -101,6 +104,10 @@
     protected final String connectionUrl;
     protected final Properties connectionProperties;
     protected final String identifierQuote;
+    private final boolean mapTableNames;
+
+    private Map<String, Map<String, String>> schemaTableMapping = new HashMap<>();
```

Comment:
You know, looking at the MySQL specific connector would have been smart to look at for the tests. Thanks for pointing that out, and I'll look at LoadingCache.